pledgie/pledgie.php

```diff
@@ -23,7 +24,7 @@ function pledgie_uninstall() {
 	Addon::unregisterHook('addon_settings_post', 'addon/pledgie/pledgie.php', 'pledgie_addon_settings_post');
 }
 
-function pledgie_addon_settings(&$a,&$s) {
+function pledgie_addon_settings(App $a,&$s) {
```

Comment:
Forgot a space after the comma.

Reply:
And new line is missing before opening curly brace.